pages/linux/cat.md

```diff
@@ -0,0 +1,20 @@
+# cat
```

Comment:
This page is in the common platform, it's not specific for Linux.

Reply:
I'm sorry :sweat_smile: removed the pages again